src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = ({ entries, updateChatEntry }) => {
+return (
+    <ul className="chat-entry local">
+    {entries.map((message) => {
+        return (
+        <ChatEntry
+            key={message.id}
+            id={message.id}
+            sender={message.sender}
+            body={message.body}
+            timeStamp={message.timeStamp}
+            liked={message.liked}
+            updateChat={updateChatEntry}
+        />
+    );
+})}
+    </ul>
+);
+};
+
+    ChatLog.propTypes = {
```

Comment:
As with `ChatEntry`, these _would_ probably all be required, but we could have left some of them not-required due to the tests. We'd also want to include a proptype for `id` under the entries shape.